apps/catalog-admin/pages/manage/index.tsx

```diff
@@ -0,0 +1,13 @@
+import {Card} from '@catalog-frontend/ui';
+import styles from './manage.module.css';
+
+export const ManagePage = () => {
+  return (
+    <div className={styles.container}>
+      <Card title='Kodeliste' body='Administrer kodeliste' />
```

Comment:
Kanskje lurt å defienre og hente strengene fra språkfilen? Noe sånt:
```suggestion
      <Card title={localization.admin.codelist} body={localization.admin.manageCodeList} />
```

Reply:
Skal catalog-admin og concept-catalog bruke samme språkfil? @pooriamehregan @jeffreiffers